packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -1378,12 +1404,11 @@ class ProxyDomain extends Domain {
     for (final Socket connection in _forwardedConnections.values) {
       await connection.close();
     }
-    await _tempDirectory?.delete(recursive: true);
+    // await _tempDirectory?.delete(recursive: true);
   }
 
-
   Directory _tempDirectory;
-  Directory get tempDirectory => _tempDirectory ??= globals.fs.systemTempDirectory.createTempSync('flutter_tool_daemon.');
+  Directory get tempDirectory => _tempDirectory ??= globals.fs.systemTempDirectory.childDirectory('flutter_tool_daemon')..createSync();
```

Comment:
what's the difference here?

Reply:
I've changed it to use the same temp directory every time the app runs, so that we could reuse the file that is transferred from the previous run.

Reply:
ahh, makes sense.

Reply:
could you leave a breadcrumb comment, lest someone revert this in a "cleanup"?

Reply:
Good point about leaving a comment, just added. Thanks!